scarb/src/bin/scarb/commands/init.rs

```diff
@@ -24,7 +25,14 @@ pub fn run(args: InitArgs, config: &Config) -> Result<()> {
             } else {
                 VersionControl::Git
             },
-            snforge: args.snforge,
+            snforge: matches!(
+                if let Some(test_runner) = args.test_runner {
```

Comment:
Maybe https://doc.rust-lang.org/core/option/enum.Option.html#method.unwrap_or_else? 🤔

Reply:
You can't use `?` in closures and `unwrap_or_else` uses them

Reply:
Hmm, maybe something like this
```
Ok(args.test_runner).transpose().unwrap_or_else(ask_for_test_runner)?
```

Reply:
Looks clean 😄, I will add this tomorrow